Gateway/Request/CheckoutDataBuilder.php

```diff
@@ -319,28 +329,29 @@ protected function getOpenInvoiceData($order): array
         ) {
             $shippingAmountCurrency = $this->chargedCurrency->getQuoteShippingAmountCurrency($cart);
 
-            $priceExcludingTax = $shippingAmountCurrency->getAmount();
+            $formattedPriceExcludingTax = $this->adyenHelper->formatAmount(
+                $shippingAmountCurrency->getAmount(),
+                $currency
+            );
+
+            $formattedPriceIncludingTax = $this->adyenHelper->formatAmount(
+                $shippingAmountCurrency->getAmountIncludingTax(),
```

Comment:
is this function calculates the tax depending on the settings in the magento config panel? (tax added to cart / item etc? )

Reply:
No, not really a calculation. It will only get $quote->getShippingAddress()->getShippingInclTax() which contains the right amount regardless of the config selected. Same for $quoteItem->getPriceInclTax() for regular line items